provider/keycloak_openid_client_authorization_scope.go

```diff
@@ -0,0 +1,135 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"strings"
+)
+
+func resourceKeycloakOpenidClientAuthorizationScope() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationScopeCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationScopeRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationScopeDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationScopeUpdate,
+		// This resource can be imported using {{realm}}/{{resource_server_id}}. The Client ID is displayed in the GUI
```

Comment:
This comment doesn't seem correct.  You can just remove it if you want.

Reply:
Done